src/CONST/index.ts

```diff
@@ -2987,6 +2987,14 @@ const CONST = {
             NAME_USER_FRIENDLY: {
                 uber: 'Uber for Business',
             },
+            UBER_EMPLOYEE_STATUS: {
```

Comment:
I think these are wrong? Look at here https://developer.uber.com/docs/employees/references/api/v1/get-v1-business-organizations-id-employee the values for status are:
```
CREATED
INVITED
LINKED_PENDING_APPROVAL
LINKED
SUSPENDED
DELETED
```